### PR TITLE
Feature/secure message

### PR DESCRIPTION
**Pull request by mg-4xxi**

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -49,6 +49,7 @@ MAILER_URL=null://localhost
 #DELIVERY_ADDRESS=
 #DB_VERSION=9.6
 #WEB_CLIENT_URL=
+#TEST_DATABASE_URL=pgsql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db_test:5432/${POSTGRES_DB}
```
Comment (beshkenadze) [inline .env.dist]: We need to remove it.

File: docker-compose.yml [hunk 2]
```diff
@@ -37,6 +37,15 @@ services:
             - POSTGRES_DB
             - POSTGRES_USER
             - POSTGRES_PASSWORD
+    db_test:
```
Comment (beshkenadze) [inline docker-compose.yml]: Do not use dedicated DB for tests.

Comment (mg-4xxi) [reply docker-compose.yml]: removed

File: .env.dist (hunk 1 above)
Comment (mg-4xxi) [reply .env.dist]: removed

**Pull request merged**